projects.yml

```diff
@@ -4572,3 +4572,9 @@ calva:
   url: https://github.com/BetterThanTomorrow/calva
   categories: [IDE Integration]
   platforms: [clj, cljs]
+
+clojure-maven-plugin:
+  name: vivid:clojure-maven-plugin
```

Comment:
This should probably be "Vivid Clojure Maven Plugin" since that's the title of the README.

Reply:
Alright let's go with your suggestion and see where it takes us